### PR TITLE
Add php7 test in userspace

### PR DESCRIPTION
**Pull request by SeroSun**
Add php7 test in qa_automation package for SLE15

### CODE REVIEW

File: tests/qa_automation/userspace_php7.pm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# G-Summary: split userspace into pieces for easy review.
+# G-Maintainer: Yong Sun <yosun@suse.com>
```
Comment (okurz) [inline tests/qa_automation/userspace_php7.pm]: needs to be updated, also line 3-4

**Comment by SeroSun** [comment]
@okurz Updated :)

File: tests/qa_automation/userspace_php7.pm [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: add php7 in userspace regression test.
```
Comment (okurz) [inline tests/qa_automation/userspace_php7.pm]: The summary should describe the goal of the test not repeat your git commit message

**Comment by SeroSun** [comment]
Agree, and updated @okurz

**Comment by okurz** [comment]
yep, looks good :)

**Pull request merged**